docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+shipit:
+  build: ./
+  command: python kickoff-web.py -d sqlite:///update.db --username=admin --password=password --host=0.0.0.0 --port=5000 --verbose
```

Comment:
Using sqlite without a bind mount will give you a non-persistent database AFAIK. I strongly recommend using a separate container with MySQL instead, since that's what's used in production. Docker-compose makes this very easy, eg: https://github.com/mozilla/balrog/blob/master/docker-compose.yml#L30